tests/Propulsion.Kafka.Integration/ConsumersIntegration.fs

```diff
@@ -62,8 +62,9 @@ module Helpers =
             Task.Delay(delay).ContinueWith(fun (_:Task) -> c.Stop()) |> ignore
 
     type TestMeta = { key: string; value: string; partition: int; offset : int64 }
-    let mapConsumeResult (x: ConsumeResult<_,_>) : KeyValuePair<string,string> =
-        KeyValuePair(x.Key, JsonConvert.SerializeObject { key = x.Key; value = x.Value; partition = Bindings.partitionId x; offset = let o = x.Offset in o.Value })
+    let mapParallelConsumeResult (x: ConsumeResult<_,_>) : KeyValuePair<string,string> =
```

Comment:
`consumeResultToKeyValuePair`?

Reply:
it aligns with a name futher down (mapStreamsConsumeResult) so will hold for now